security/mkcert/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+
+github.setup        FiloSottile mkcert 1.1.0
+github.tarball_from releases
+
+platforms           darwin
+categories          security devel
+license             AS-IS
```

Comment:
Is this a valid license name? (License names aren't arbitrary, we have a set that we use in order to do things like automatically finding license conflicts.)